lib/shims/eq/R2015b/newline.m

```diff
@@ -0,0 +1,9 @@
+function result = newline
+
+% NEWLINE Create newline character
+% NEWLINE Introduced in R2016b
+% https://www.mathworks.com/help/matlab/ref/newline.html
+
+result = sprintf('\n');
```

Comment:
Can you stash this in a `persistent` variable, for performance? `sprintf` is a nontrivial function call.

Reply:
PR is updated with this feedback implemented.